src/index.d.ts

```diff
@@ -86,6 +86,13 @@ interface TabsProps {
    * @param index
    */
   onChange(index: number): void;
+
+  /**
+   * Optional keyboard tap behaviour for the ScrollView.
+   * See: https://facebook.github.io/react-native/docs/scrollview#keyboardshouldpersisttaps
+   * Default 'none'.
+   */
+  keyboardShouldPersistTaps?: string;
```

Comment:
Instead of string use a union type.


`keyboardShouldPersistTaps?: "always" | "never" | "handled";`

Reply:
cheers for the review! still getting my head around typescript